app/models/webhook.rb

```diff
@@ -33,11 +33,11 @@ class Webhook < ApplicationRecord
   validates :secret, presence: true, length: { minimum: 12 }
   validates :events, presence: true
 
-  validate :validate_events
+  validate :events_validation_error, if: :invalid_events?
   validate :validate_permissions
   validate :validate_template
 
-  before_validation :strip_events
+  normalizes :events, with: ->(events) { events.filter_map(&:strip).filter_map(&:presence) }
```

Comment:
Why this change? Also it seems to run counter to #27744

Reply:
Yes it does ... I guess the rule is just for `map` and not `filter_map` as well. Either that or this was exluded from cop somehow. Either way, pushed a change to keep prior style.